### PR TITLE
Column config fixups

### PR DESCRIPTION
**Pull request by kevinhinterlong**
## Description
Resolves some small issues we've found in the column config
- Font family not being applied so bold doesn't work as expected
- Searching a metrics parameters is not working
- Column relabeling was visible in the UI, but not functional so we've hidden it for now

## License

I confirm that this contribution is made under an MIT license and that I have the authority necessary to make this contribution on behalf of its copyright owner.

### CODE REVIEW

File: packages/reports/addon/templates/components/navi-column-config/metric.hbs [hunk 1]
```diff
@@ -7,7 +7,8 @@
       @triggerId="{{this.classId}}-columnParameter" 
       @dropdownClass="navi-column-config-metric__parameter-dropdown"
       @selected={{this.currentParameter}}
-      @searchPlaceholder="Type to search"
+      @searchField="id"
+      @searchPlaceholder="Type to search by id"
```
Comment (jkusa) [inline packages/reports/addon/templates/components/navi-column-config/metric.hbs]: Can this be `Search ${paramName}`?

File: packages/reports/addon/templates/components/navi-column-config/metric.hbs [hunk 2]
```diff
@@ -7,7 +7,8 @@
       @triggerId="{{this.classId}}-columnParameter" 
       @dropdownClass="navi-column-config-metric__parameter-dropdown"
       @selected={{this.currentParameter}}
-      @searchPlaceholder="Type to search"
+      @searchField="id"
```
Comment (jkusa) [inline packages/reports/addon/templates/components/navi-column-config/metric.hbs]: Why `id over `description`?

Comment (kevinhinterlong) [reply packages/reports/addon/templates/components/navi-column-config/metric.hbs]: We can change it to be description in both places, although id's being unique makes them nice for searching
https://github.com/yahoo/navi/blob/950c05b03d3d05a8c64ca8eee56cde79148bc852/packages/reports/addon/templates/components/metric-config.hbs#L18

File: packages/reports/addon/templates/components/navi-column-config/metric.hbs (hunk 1 above)
Comment (kevinhinterlong) [reply packages/reports/addon/templates/components/navi-column-config/metric.hbs]: What does paramName refer to, `paramName = Revenue` or `paramName = description`?

File: packages/reports/addon/templates/components/navi-column-config/metric.hbs (hunk 2 above)
Comment (jkusa) [reply packages/reports/addon/templates/components/navi-column-config/metric.hbs]: I agree about unique ID values. Just like most dimensions, users like to search by nice names. I don't think users will not really care about the param ID values.

**Pull request merged**